src/main/java/HandleTruth.java

```diff
@@ -33,5 +38,11 @@ public static void wordCount() {
         for (Map.Entry<Integer, Set<String>> entry : sortedMap.entrySet()) {
             System.out.println(entry);
         }
+
+        if(mapType) {
+            return wordMap;
```

Comment:
wordMap is an internal result. the way it's set up here is a bit weird.

instead, refactor the counting and ordering into two functions and write tests against both instead of trying to do it in this one function.